src/borg/archiver/create_cmd.py

```diff
@@ -187,6 +187,11 @@ def create_inner(archive, cache, fso):
                 if args.progress:
                     archive.stats.show_progress(final=True)
                 archive.stats += fso.stats
+                try:
+                    archive.stats.rx_bytes = repository.rx_bytes
+                    archive.stats.tx_bytes = repository.tx_bytes
+                except AttributeError:
+                    pass  # We are working with a local repo
```

Comment:
yeah, would work like that.

you could also use `getattr(repository, 'rx_bytes', 0)`.